pkg/xray/datasource/datasource_xray_curation_condition_test.go

```
@@ -0,0 +1,880 @@
+
+		rawPageNum := r.URL.Query().Get("page_num")
+		pageNum, err := strconv.Atoi(rawPageNum)
+		if err != nil {
+			t.Errorf("request `page_num` = %q, want an integer", rawPageNum)
+		}
+		requestedPages = append(requestedPages, pageNum)
+
+		status, body := handler(pageNum)
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(status)
+		fmt.Fprint(w, body)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)